rest/data/host_create.go

```diff
@@ -183,14 +183,19 @@ func makeProjectAndExpansionsFromTask(ctx context.Context, settings *evergreen.S
 	if err != nil {
 		return nil, nil, errors.Wrap(err, "getting GitHub OAuth token from admin settings")
 	}
-	appToken, err := settings.CreateInstallationToken(ctx, v.Owner, v.Repo, nil)
+	pRef, err := model.FindBranchProjectRef(t.Project)
+	if err != nil {
+		return nil, nil, errors.Wrapf(err, "finding project ref '%s'", t.Project)
+	}
```

Comment:
This needs to check if `pRef` is nil since the FindOne methods in Evergreen generally `return nil, nil`.